internal/daemon/controller/interceptor_test.go

```diff
@@ -724,13 +724,13 @@ func Test_aliasResolutionInterceptor(t *testing.T) {
 			name:            "aliasable request with unknown alias",
 			req:             &pbs.GetTargetRequest{Id: "not.a.registered.alias"},
 			wantModifiedReq: &pbs.GetTargetRequest{Id: "not.a.registered.alias"},
-			errorContains:   "resource alias not found with value",
+			errorContains:   "Forbidden",
```

Comment:
Is it worth checking that the error is _exactly_ `"Forbidden"` in this and the other case? It doesn't help if we return different error strings because of wrapping or something.